src/cpp/garbage-collector.cpp

```diff
@@ -7,10 +7,12 @@
 
 namespace effil {
 
+static const size_t MinimumSizeLeft = 100;
```

Comment:
You can omit `static`. Better use constexpr instead of `static const`.
Also naming convention for consts is `MINIMUM_SIZE_LEFT`.

Reply:
Yes, ofcourse it should be constexpr. Who the hell use that type of names for constants?))